README.md

```diff
@@ -230,6 +239,9 @@ Invalid argument: Second argument 2.3 is not an integer
   * This is actually a good thing, in light of [this](#kinds-of-errors) and [[8]](#resources).
```

Comment:
Also this link doesn't work plus there's one more references to [8] that needs to be fixed. #resources doesn't exist anymore, it's #external-resources instead